### ISSUE TITLE
Bug: undefined parent type and label at v.3.0 notifications

### ISSUE THREAD

**Issue by gbonaspetti** [tracim/tracim#3586]
## Description and expectations

Notifications of type `content.created.comment` that were created in version 3.0 of Tracim have neither `parent_label` nor `parent_type`, this causes the following bug in the display and sends the user to a URL of type `.../ui/workspaces/{{workspace_id}}/contents/undefined/{{content_id}}`.

- Display bug:
![Capture d’écran de 2020-09-15 09-56-51](https://user-images.githubusercontent.com/35422692/93194605-918f1200-f748-11ea-88c3-6ae3da0488e8.png)

### Version information

- Tracim version: v3.1
- Web browser: any
- Version: any
- Device / System: any

## Expected
- [ ] If an element is `undefined` should be displayed as _unknown_
- [ ] When click, should display a flash warning message (see translation table)

## Translations

|English|French|Portuguese|
|--|--|--|
|unknown|inconnu|desconhecido|
|Sorry, this content is unknown|Désolé, ce contenu est inconnu|Desculpe, este conteúdo é desconhecido|

________________________________________________

## Diagnostic

This properties doesn't existed at Tracim v.3.0.

## Prerequisites

None

**Comment by raphj** [comment]
I don't think we should show "Sorry, this content is unknown", but rather something like "This notification was created with an older version of Tracim. We cannot open this content from the notification".
